### ISSUE TITLE
Strange issue with "quasar/src/css/normalize.sass" and SASS processor

### ISSUE THREAD

**Issue by tmcdos** [quasarframework/quasar#6933]
**Describe the bug**
I am trying to develop a Vue.js application with Quasar without using the Quasar CLI. I am facing a strange warning from the SASS compiler
```
WARNING on line 106, column 1 of normalize.sass:
This selector doesn't have any properties and won't be rendered.
106 │ ┌ button,
107 │ └ input, /* 1 */
```
It does not make sense as the code seems perfectly valid. However, if I remove the comment from line 107 - the warning goes away.

**Codepen/jsFiddle/Codesandbox (required)**
Fork a Codepen (https://codepen.quasar.dev) or a jsFiddle (https://jsfiddle.quasar.dev) or a Codesandbox (https://codesandbox.quasar.dev) and hit save then copy-paste link here.

**To Reproduce**
1. Create an empty folder
2. Copy `quasar/src/css/normalize.sass` into the above folder
3. Create a file `test.js` inside the above folder with the following content
```js
require('sass').renderSync({file: "normalize.sass"});
```
4. Run `npm install sass` inside the above folder
5. Run `node test.js` inside the above folder

**Expected behavior**
There should be no errors or warnings when processing the SASS file.

**Screenshots**
![shot-1](https://user-images.githubusercontent.com/19253115/80863175-235b1e80-8c83-11ea-9221-3d252eab182b.png)

**Platform (please complete the following information):**
Quasar: 1.9.16
OS: Windows 7 SP1 x64
Node: 13.13.0
NPM: 6.13.6
Yarn: None
Browsers: None
iOS: None
Android: None
Electron: None

**Additional context**

**Comment by pdanpdan** [comment]
Hello. Sorry, this has nothing to do with Quasar.

**Comment by tmcdos** [comment]
If you change line 107 like this
```
input /* 1 */,
```
the warning goes away.

**Comment by rstoenescu** [comment]
@tmcdos Enhancement will be available in 1.10.1
